content/docs/ops/service-disruption-guide.md

```diff
@@ -0,0 +1,58 @@
+---
+menu:
+  docs:
+    parent: policies
+title: Service disruption guide
+---
+
+This guide outlines how we handle communications for service disruptions during East Coast business hours.
+
+Not a guide for:
+
+* Hours outside of East Coast business hours -- out of scope for this document.
+* Security incidents -- switch to [security incident guide]({{< relref "docs/ops/security-ir.md" >}}).
+* Major service disruptions (more than 30 minutes of unexpected downtime or significantly reduced service) -- switch to [contingency plan]({{< relref "docs/ops/contingency-plan.md" >}}).
```

Comment:
30 minutes seems like a low threshold for going from disruption to contingency. We should probably discuss if a bigger number makes sense because I think some of our recent outages would have been over that threshold.